src/main/pages/che-6/workspace-admin/projects.adoc

```diff
@@ -35,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
  ]
 ----
 
-Once a project is saved into a workspace config, the IDE will check if the project exists on a file system, and if not, import it using `source.location` URL. It happens at IDE initialization stage.
+Once a project is saved into a workspace configuration, the IDE will check if the project exists on a file system.  If the project does not exist on the file sytem, you can import it using the `source.location` URL. This happens during the IDE initialization stage.
 
-You can add Git projects, remotely hosted archives, your GitHub projects or project samples that Eclipse Che offers out of the box. Project import tools can be found both in User Dashboard (when creating a new workspace or editing an existing one) on in the IDE, Workspace menu.
+You can add the following projects:
 
-image::workspaces/projects.png[]
+* Git projects 
+* remotely hosted archives
+* GitHub projects 
+* example projects provided by Eclipse Che 
 
-[id="sub-projects"]
-== Sub-Projects
+Project import tools can be found on the *User Dashboard* when you are creating a new workspace or editing an existing workspace on the IDE. Project import tools can also be found on the *Workspace* menu.
```

Comment:
Regarding "on the IDE" -- did you mean "in the IDE" here? Using "on" seems awkward to me, but I may not be understanding the context.